airflow/config_templates/default_airflow.cfg

```diff
@@ -678,6 +678,9 @@ git_dags_folder_mount_point =
 git_ssh_key_secret_name =
 git_ssh_known_hosts_configmap_name =
 
+# To give git_sync a secret containing git credentials, use this
```

Comment:
Could you extend the comment to say what fields are read from the secret?

Reply:
Definitely.  How does this look @ashb ?